ansible/roles/ocp-workload-fsi-cc-dispute-demo/tasks/pre_workload.yml

```diff
@@ -0,0 +1,44 @@
+---
+
+# - name: Add user to developer group (allowed to create projects)
+#   shell: "oadm groups add-users {{item}} {{ocp_username}}"
+#   register: groupadd_register
+#   with_items: "{{ocp_user_groups}}"
+#   when: ocp_username is defined and ocp_user_groups is defined
+#
+# - name: test that command worked
+#   debug:
+#     var: groupadd_register
+#     verbosity: 2
+
+- name: Show path
+  shell: "echo $PATH"
+
+
+- name: Create user Quota - clusterresourcequota
+  shell: |
```

Comment:
You don't need `shell` module here (no shell feature used), command is enough. Also, multiline YAML `>` notation is more convenient in this case:

```yaml
- command: >-
    mycommand
    arg1
    arg2
```

instead of 

```yaml
- shell: |
    mycommand \
    arg1 \
    arg2
```

Reply:
done